unstructured/cleaners/translate.py

```diff
@@ -0,0 +1,82 @@
+from typing import List, Optional
+import warnings
+
+import langdetect
+from transformers import MarianMTModel, MarianTokenizer
+
+from unstructured.staging.huggingface import chunk_by_attention_window
+from unstructured.nlp.tokenize import sent_tokenize
+
+
+def _get_opus_mt_model_name(source_lang: str, target_lang: str):
+    """Constructs the name of the MarianMT machine translation model based on the
+    source and target language."""
+    return f"Helsinki-NLP/opus-mt-{source_lang}-{target_lang}"
+
+
+def _validate_language_code(language_code: str):
+    if not isinstance(language_code, str) or len(language_code) != 2:
+        raise ValueError(
+            f"Invalid language code: {language_code}. Language codes must be " "two letter strings."
```

Comment:
Is there a reason this is broken up into two strings? Did `black` join two lines or something?

Reply:
Yeah it was a long string and `black` must have combined them. I'll fix that